frame/election-provider-multi-phase/src/mock.rs

```diff
@@ -297,6 +297,8 @@ parameter_types! {
 	pub static MockWeightInfo: MockedWeightInfo = MockedWeightInfo::Real;
 	pub static MaxElectingVoters: VoterIndex = u32::max_value();
 	pub static MaxElectableTargets: TargetIndex = TargetIndex::max_value();
+
+	#[derive(Debug)]
```

Comment:
I was forced to add this `Debug bound` I suspect it was because it was resolved by rustc before as this was part of the `Config trait`.

Only relevant for tests anyway so should be a big deal